optuna/multi_objective/study.py

```diff
@@ -356,14 +358,15 @@ def user_attrs(self) -> Dict[str, Any]:
         return self._study.user_attrs
 
     @property
+    @deprecated_func("3.1.0", "6.0.0")
```

Comment:
MultiObjectiveStudy` itself is deprecated and will be removed in v4.0.0 release. How about removing `deprecated_func` here?

Reply:
Oh, I forgot it. I removed this `deprecated_func`.